### PR TITLE
feat: removes dependency on fast-foundation templates and classes.

### PR DESCRIPTION
**Pull request by eljefe223**
## New Behavior
Removes dependency on fast-foundation classes and templates.

## Related Issue(s)

- Fixes #

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/12da47698c9121c0345c2f0db481cbae7a91b66e'>12da47698c9121c0345c2f0db481cbae7a91b66e</a></sub>
<div id="bundleSizeComment9423"/>

File: change/@fluentui-web-components-b0669382-c509-4801-bd44-9849e8570a92.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-web-components-b0669382-c509-4801-bd44-9849e8570a92.json:1]: ## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30090&commitId=9535c4ad17188efbcb3e81a7b9aff6b113184840&env=prod&clientType=FLUENTUI)


  <details><summary><b>Tabs</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tabs.Size Medium Vertical.normal.chromium.png|30953| Changed|
|Tabs.Size Small Vertical - Dark Mode.normal.chromium.png|3377| Changed|
|Tabs.Size Large Vertical - Dark Mode.normal.chromium.png|6551| Changed|
|Tabs.Size Medium Vertical - Dark Mode.normal.chromium.png|5088| Changed|
|Tabs.Vertical.normal.chromium.png|30953| Changed|
|Tabs.Size Large Vertical.normal.chromium.png|39762| Changed|
|Tabs.Size Small Vertical.normal.chromium.png|19375| Changed|
|Tabs.Navigation Vertical.chromium.png|641| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Review by mohamedmansour — APPROVED** [review]

**Review by chrisdholt — APPROVED** [review]

File: change/@fluentui-web-components-b0669382-c509-4801-bd44-9849e8570a92.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-web-components-b0669382-c509-4801-bd44-9849e8570a92.json:1]: ## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30090&commitId=2a602410c0e2490fcc47364f519d55acbd6452bb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

File: change/@fluentui-web-components-b0669382-c509-4801-bd44-9849e8570a92.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-web-components-b0669382-c509-4801-bd44-9849e8570a92.json:1]: ## [🕵 fluentui-web-components-v3 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30090&commitId=01faebee3ac3316c3798a8c026613a431a2b0bfb&env=prod&clientType=FLUENTUI)


  <details><summary><b>Slider</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Slider.Slider Steps Vertical.normal.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Comment by fabricteam** [comment]
## 🕵 fluentui-web-components-v3 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_web-components"/>

**Pull request merged**